sdks/java/core/src/test/java/org/apache/beam/sdk/io/CountingInputTest.java

```diff
@@ -71,14 +71,35 @@ public static void addCountingAsserts(PCollection<Long> input, long start, long
   public void testBoundedInput() {
     Pipeline p = TestPipeline.create();
     long numElements = 1000;
-    PCollection<Long> input = p.apply(CountingInput.upTo(numElements));
+    PCollection<Long> input = p.apply(Read.from(CountingSource.upTo(numElements)));
```

Comment:
I fixed this in https://github.com/apache/incubator-beam/commit/5169e492578a3759e20b50e36ace61bc86636ad2